File: HELLO_PYTHON/day06/mysql_insert.py
import pymysql
 
conn = pymysql.connect(host='localhost', user='root', password='python',
                       port=3305, db='python', charset='utf8')
 
curs = conn.cursor()
sql = """
        insert into emp
        (e_id, e_name, gen, addr)
        values 
        (%s, %s, %s, %s)
"""
         
curs.execute(sql, ('4', '4', '4', '4'))
conn.commit()

curs.close()
conn.close()

File: HELLO_NODE/mysql_insert.js
const mysql = require('mysql');
const connection = mysql.createConnection({
	host: 'localhost',
	port: 3305,
	user: 'root',
	password: 'python',
	database: 'python'
});

connection.connect();
console.log("start");
var sql = `
	insert into emp
		(e_id, e_name, gen, addr)
	values
		('3', '3', '3', '3')
`;

connection.query(sql, (error, result) => {
	console.log(result.affectedRows);
	console.log("result");
});

connection.end();
console.log("end");